docker/build_scripts/build.sh

```diff
@@ -87,6 +71,10 @@ yum -y install \
     bison \
     bzip2 \
     cmake28 \
+    devtoolset-8-binutils \
```

Comment:
What about the other packages which used to be installed, like `runtime`, `libstdc++` and `libquadmath-devel`?

Reply:
They are dependencies of these ones and are still installed.